.ci/bintray-release.json

```diff
@@ -0,0 +1,26 @@
+{
+  "files": [
+    {
+      "includePattern": "build/(Seaglass-.*.tar.gz)",
+      "uploadPattern": "Seaglass-VERSION_NAME_VALUE.tar.gz"
+    }
+  ],
+  "package": {
+    "desc": "A truly native Matrix client for macOS",
+    "issue_tracker_url": "https://github.com/neilalexander/seaglass",
+    "licenses": [
+      "GPL-3.0"
+    ],
+    "name": "Seaglass",
+    "subject": "aaronraimist",
```

Comment:
- [ ] Create your Bintray account, repo, and project, then change this to your username